### DISCUSSION TITLE
Emissions from tropical dams

### DISCUSSION THREAD

**Discussion by levinche** [electricitymaps/electricitymaps-contrib#4895] [Ideas]
I see the emissions of Brazilian tropical dams electricity are quite low: 24 g CO2eq/kWh as per IPCC 2014. This does not reflect the reality of tropical conditions with immense areas of submerged forest that release great quantities of methane while decaying under water.

This is explained by several scientific publications listed below, demonstrating that the default figures from IPCC 2014 are not relevant for tropical reservoir dams. We are talking of a factor 20 to 300 of error, so this is not only significant: it is of a crucial  importance if we want to disclose meaningful and accurate values for tropical dams in general, and for Brazil in particular.

I can provide some more details if needed. Let me know how I can help.

- de Faria FAM, Jaramillo P, Sawakuchi HO, et al (2015) Estimating greenhouse gas emissions from future Amazonian hydroelectric reservoirs. Environ Res Lett 10:124019. doi: 10.1088/1748-9326/10/12/124019
- Farrèr C (2007) Hydroelectric Reservoirs - the Carbon Dioxide and Methane Emissions of a “Carbon Free” Energy Source. Swiss Federal Institute of Technology in Zurich (ETHZ)
- Fearnside PM (2011) Greenhouse gas emissions from hydroelectric dams in tropical forests. Encycl. Energy. John Wiley & Sons Publishers, New York, U.S.A., New York, United States, p 20
- Fearnside PM (2015) Emissions from tropical hydropower and the IPCC. Environ Sci Policy 50:225–239. doi: 10.1016/j.envsci.2015.03.002
- Kemenes A, Forsberg BR, Melack JM (2011) CO2 emissions from a tropical hydroelectric reservoir (Balbina, Brazil). J Geophys Res 116:1–11. doi: 10.1029/2010JG001465

**Comment by unitrium** [comment]
Hi, thanks for the good feedback!
This is very interesting indeed. We have recently launched a feature called regional emission factors. This enables us to adjust the carbon intensity of production modes depending on local conditions.

It would be great to integrate the numbers of those publication into our calculations for Brazil and maybe other tropical regions.
You can have a look here at how we integrate regional emission factors:
https://github.com/electricitymaps/electricitymaps-contrib/wiki/Emission-factors

At the moment we only have done this for Europe and the US, it's on our roadmap for next year to extend it to several regions. We would need to take a look at those publications to extract the regional emission factor for each zone in Brazil.

**Comment by unitrium** [comment]
Basically what we would need is for each zone in Brazil know the emission factor for hydro power.
So that would be:
- North
- North East
- Central
- South

**Reply by levinche** [reply]
Thanks for your answer!
It is definitely doable, if not already done and published somewhere. I would need some time to dig into it, and I don't have much at the moment, but I would be glad to help.
